frontend/package.json

```diff
@@ -61,6 +62,7 @@
     "ember-intl": "^4.4.0",
     "ember-islands": "^1.5.1",
     "ember-load-initializers": "^2.1.2",
+    "ember-markdown-html": "^0.0.1",
```

Comment:
We don't need both addons right? We're using the showdown addon right? Looks better